src/tests/test_davidson.cc

```diff
@@ -68,7 +68,7 @@ BOOST_AUTO_TEST_CASE(davidson_full_matrix_fail) {
 class TestOperator : public MatrixFreeOperator {
```

Comment:
We should add a test with OperatorBlock as well no ?

Reply:
Yes we should add a blockoperator test